### PR TITLE
fix: update the project after task deletion so that the % completed s…

### PR DESCRIPTION
**Pull request by AfshanKhan**
…hows correct value

Calling update project after deleting a task so that the correct value of % completed is reflected in the project.

Backport to version-12 to be created.

### CODE REVIEW

**Review by Anurag810 — DISMISSED** [review]

**Comment by Anurag810** [comment]
Please resolve conflicts

**Comment by AfshanKhan** [comment]
> Please resolve conflicts

@Anurag810 resolved and pushed

File: erpnext/patches.txt [hunk 1]
```diff
@@ -708,4 +708,6 @@ erpnext.patches.v13_0.move_doctype_reports_and_notification_from_hr_to_payroll #
 erpnext.patches.v13_0.move_payroll_setting_separately_from_hr_settings #22-06-2020
 erpnext.patches.v13_0.check_is_income_tax_component #22-06-2020
 erpnext.patches.v12_0.add_taxjar_integration_field
+erpnext.patches.v12_0.fix_percent_complete_for_projects
+erpnext.patches.v13_0.fix_percent_complete_for_projects
```
Comment (nabinhait) [inline erpnext/patches.txt]: 2 patches is not required. Remove erpnext.patches.v13_0.fix_percent_complete_for_projects and delete the file.

**Review by nabinhait — CHANGES_REQUESTED** [review]

File: erpnext/patches.txt (hunk 1 above)
Comment (AfshanKhan) [reply erpnext/patches.txt]: Done

**Review by nabinhait — APPROVED** [review]

**Pull request merged**